packages/puppeteer-core/src/bidi/Input.ts

```diff
@@ -699,3 +705,47 @@ export class BidiTouchscreen extends Touchscreen {
     ]);
   }
 }
+/**
+ * @internal
+ */
+export class BidiTouchscreen extends Touchscreen {
+  #page: BidiPage;
+  #idRepository = new TouchPointIdRepository();
+  #touches: BidiTouch[] = [];
+
+  constructor(page: BidiPage) {
+    super();
+    this.#page = page;
+  }
+
+  override async touchStart(x: number, y: number): Promise<Touch> {
+    const id = this.#idRepository.getId();
+    const properties: Bidi.Input.PointerCommonProperties = {
+      width: 0.5 * 2, // 2 times default touch radius.
+      height: 0.5 * 2, // 2 times default touch radius.
+      pressure: 0.5,
+      altitudeAngle: Math.PI / 2,
+    };
+    const touch = new BidiTouch(this.#page, id, x, y, properties);
+    await touch.start();
+    this.#touches.push(touch);
+    return touch;
+  }
+
+  override touchMove(x: number, y: number): Promise<void> {
+    const touch = this.#touches[0];
+    if (!touch) {
+      return Promise.resolve();
```

Comment:
Nit: No need for `Promise.resolve` if you use async await, that is the preferred syntax we have.